src/sonic-config-engine/tests/test_minigraph_case.py

```diff
@@ -82,15 +82,39 @@ def test_minigraph_interfaces(self):
     def test_minigraph_vlans(self):
         argument = '-m "' + self.sample_graph + '" -p "' + self.port_config + '" -v VLAN'
         output = self.run_script(argument)
+        expected = {
```

Comment:
Looks like some strings here are unicode, whereas others arent. Is there a reason for this?

Reply:
No, it isn't needed. Updated